src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -287,6 +287,16 @@ public IActionResult GetAdminToken()
             return Ok(requestHeaderToken);
         }
 
+        [HttpPost]
```

Comment:
Need to double check that this is expected to be a POST request.

Reply:
Looking at the platform code, this needs to be GET request.
Phew! That have been a tiny disaster.

Reply:
Let's make it GET and POST to be safe.